### PR TITLE
3.0 preparation: jakarta packages

### PR DESCRIPTION
**Pull request by tomas-langer**
Upgraded all relevant (and available) dependencies to Jakarta namespaces.
Upgraded all MP specification version to latest available (usually an RC)
Disabled all failing tests.
Added comments to some MP tests with details of failure (lot caused by OpenAPI).

Any comment with value `3.0.0-JAKARTA` means that something is disabled/commented out because it fails now.

Next steps:
- implement all MP specifications
- re-enable all relevant tests 
- update native image support
- re-enable all tests

### CODE REVIEW

File: integrations/cdi/jpa-cdi/src/main/xsd/persistence_3_0.xsd [hunk 1]
```diff
@@ -0,0 +1,342 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2008, 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (danielkec) [inline integrations/cdi/jpa-cdi/src/main/xsd/persistence_3_0.xsd:3]: All rights reserved.

Comment (tomas-langer) [reply integrations/cdi/jpa-cdi/src/main/xsd/persistence_3_0.xsd]: Fixed

File: microprofile/messaging/core/src/test/java/io/helidon/microprofile/messaging/connector/ConnectorTest.java [hunk 2]
```diff
@@ -19,17 +19,18 @@
...
+import jakarta.enterprise.inject.spi.DeploymentException;
+import jakarta.enterprise.util.AnnotationLiteral;
 import org.eclipse.microprofile.reactive.messaging.spi.Connector;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Disabled
```
Comment (danielkec) [inline microprofile/messaging/core/src/test/java/io/helidon/microprofile/messaging/connector/ConnectorTest.java:33]: TODO: 3.0.0 JAKARTA

Comment (tomas-langer) [reply microprofile/messaging/core/src/test/java/io/helidon/microprofile/messaging/connector/ConnectorTest.java:33]: Could you kindly write this one down? I would prefer that to adding another commit right now.

**Review by danielkec — APPROVED** [review]

**Review by romain-grecourt — APPROVED** [review]

**Pull request merged**